applications/RomApplication/python_scripts/ConvectionDiffusionROMsolver.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+
+# Import base class file
+from KratosMultiphysics.ConvectionDiffusionApplication import convection_diffusion_base_solver
+import KratosMultiphysics.RomApplication as romapp
+
+def CreateSolver(main_model_part, custom_settings):
```

Comment:
The create method of the solver must have a model and a parameters object as arguments.

Reply:
The format is taken from the convection_diffusion_base_solver.py